<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1175</width>
    <height>901</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Criador de Layout</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>541</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btn_load_pdf">
       <property name="text">
        <string>Carregar PDF de Amostra</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_clear_selection">
       <property name="text">
        <string>Limpar Seleção</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_save_layout">
       <property name="text">
        <string>Salvar Layout</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>94</y>
      <width>1141</width>
      <height>761</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTableWidget" name="table_widget_fields">
       <property name="columnCount">
        <number>2</number>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Nome do Campo</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Coordenadas</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="PdfViewer" name="graphics_view_pdf"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1175</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PdfViewer</class>
   <extends>QGraphicsView</extends>
   <header>src.gui.layout_builder_window</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
